### PR TITLE
add code of conduct file

### LINKED ISSUE #24: Add Code of Conduct
Hevy @Yash-Parsana ,We recommend that every repo has a code of conduct. If you don’t feel comfortable creating your own Code of Conduct from scratch we highly recommend using one of the templates provided by GitHub. 

Sample: https://github.com/Recode-Hive/Stackoverflow-Analysis/blob/main/CODE_OF_CONDUCT.md

Please assign this task to me .

### PR DESCRIPTION
**Pull request by aasthakourav20**
Fixes: #24 

Added Code of Conduct Guidelines

Closes #24

### CODE REVIEW

**Comment by aasthakourav20** [comment]
Hey  @Yash-Parsana , the issue is resolved , Kindly review it .

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+[Mayank Sharma]https://www.linkedin.com/in/mayanksharma2808/
```
Comment (Yash-Parsana) [inline CODE_OF_CONDUCT.md]: change this url to [Yash Parsana]https://www.linkedin.com/in/yash-parsana-a9176a1b2/

Comment (aasthakourav20) [reply CODE_OF_CONDUCT.md]: Hey Yash , Changes are done

**Comment by aasthakourav20** [comment]
Hey @Yash-Parsana , kindly assign this issue to me under GSSoC'24. And as my PR is merged kindly add labels like level1, level2, level3 and GSSoC levels as labels are important to get points.

**Comment by sanjay-kv** [comment]
@Yash-Parsana could you add me into the repo, I have mailed regarding this, so I can fix all labelled related issues.

**Pull request merged**